css/styles.css

```diff
@@ -71,9 +94,28 @@ body {
     left: 0;
     width: 100vw;
     height: 100vh;
-    background-color: gray;
-    opacity: 0.5;
+    background-color: rgba(0, 0, 0, .75);
```

Comment:
this is what i was forgetting when we were trying to implement the modal, and its overlay, the other day. trying to set the `opacity` property of the `overlay` class will also affect the `opacity` of all the HTML elements inside `overlay`.

instead, we can use a `background-color` with an alpha (aka opacity) value of less than `1` (in this case, `.75`). that way, we only affect the `background-color` of the `overlay`, and none of its child elements.